synfig-studio/src/gui/widgets/widget_timeslider.cpp

```diff
@@ -172,6 +174,10 @@ Widget_Timeslider::Widget_Timeslider():
 			  | Gdk::BUTTON_RELEASE_MASK
 			  | Gdk::BUTTON_MOTION_MASK
 			  | Gdk::SCROLL_MASK );
+
+	auto icon_theme = Gtk::IconTheme::get_default();
+	lower_boundary_pixbuf = icon_theme->load_icon("bound_button_icon_lower", 1);
+	upper_boundary_pixbuf = icon_theme->load_icon("bound_button_icon_upper", 1);
```

Comment:
There is no need to this "1". Or is there?

Reply:
This is the desired icon size.